sdk/Lusid.Sdk.Tests/tutorials/Instruments/CreditDefaultSwapCashFlow.cs

```diff
@@ -113,14 +113,34 @@ public void ExamplePortfolioCashFlowsForCreditDefaultSwaps()
         internal static void CreateAndUpsertRecipe(string code, string scope, ModelSelection.ModelEnum model)
         {
             // CREATE a rule for reset quotes
-            var resetRule = new MarketDataKeyRule("Equity.RIC.*", "Lusid", scope, MarketDataKeyRule.QuoteTypeEnum.Price, "mid", quoteInterval: "1Y");
+            var resetRule = new MarketDataKeyRule(
+                key: "Equity.RIC.*",
+                supplier: "Lusid",
+                scope,
+                MarketDataKeyRule.QuoteTypeEnum.Price,
+                field: "mid",
+                quoteInterval: "1Y");
+            var ratesRule = new MarketDataKeyRule(
+                key: "Rates.*.*",
+                supplier: "Lusid",
+                scope,
+                MarketDataKeyRule.QuoteTypeEnum.Price,
+                field: "mid",
+                quoteInterval: "6Y");
```

Comment:
I'd make this either 1Y or 10Y to match the equity and credit ones. If they can all be 1Y that'd be ideal.

Reply:
At least the Rates.*.* needs to be at least 6Y (5Y doesn't work).
Credit.*.*.*.* also works with 6Y.
You just want them uniform? so make them all 6Y? or want them to look round too, so make them all 10Y?

Reply:
Perhaps 10Y works if it needs at least a 5Y lookback. Not sure what mkt data it depends on but might look rounder than 6Y.

Reply:
yep, let's go with 10